R/tm_t_binary_outcome.R

```diff
@@ -504,6 +513,8 @@ tm_t_binary_outcome <- function(label,
     control$strat$method_ci, c("wald", "waldcc", "cmh", "ha", "strat_newcombe", "strat_newcombecc")
   )
   checkmate::assert_subset(control$strat$method_test, c("cmh"))
+  decorators <- normalize_decorators(decorators)
+  assert_decorators(decorators, null.ok = TRUE, "table")
```

Comment:
Please reorder these in all PRs to keep a sane order of parameters.

The initial boilerplate code was wrong and was propagated to all :-\

```suggestion
  assert_decorators(decorators, "table", null.ok = TRUE)
```

Reply:
Oh, I already merged some PR! I'll check on the remaining ones